### PR TITLE
chore: add doc-related commands to contributing and makefile

### PR DESCRIPTION
**Pull request by vdusek**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -100,6 +100,12 @@ Enable Corepack, which installs Yarn automatically:
 corepack enable
 ```
 
+Build the API reference:
+
+```sh
+./build_api_reference.sh
```
Comment (janbuchar) [inline CONTRIBUTING.md:106]:
```suggestion
make build-api-reference
```

**Review by janbuchar — APPROVED** [review]
LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (vdusek) [reply CONTRIBUTING.md:106]: it would not work as we are in website/ directory

Comment (janbuchar) [reply CONTRIBUTING.md:106]: Huh. Do we need to be there?

**Pull request merged**